themes/hive/layouts/partials/repl.html

```diff
@@ -17,7 +17,7 @@
   under the License. -->
 
 <replication class="col-md">
-    <div id="repl" data-termynal data-ty-typeDelay="40" data-ty-lineDelay="70" style="width: 830px;">
+    <div id="repl" data-termynal data-ty-typeDelay="40" data-ty-lineDelay="70" style="width: 830px; font-size: 0.9em;" >
         <span data-ty="input">jdbc:hive2://> repl dump src with (</span>
         <span data-ty="input">. . .>   'hive.repl.dump.version'= '2',</span>
```

Comment:
could you replace `. . .>` to `->` in 4 places

Reply:
Sure, i just pushed a fix.
Thanks!